hbase-zookeeper/src/test/java/org/apache/hadoop/hbase/zookeeper/TestReadOnlyZKClient.java

```diff
@@ -79,6 +84,10 @@ public class TestReadOnlyZKClient {
   private static int CHILDREN = 5;
 
   private static ReadOnlyZKClient RO_ZK;
+  static final HashedWheelTimer RETRY_TIMER = new HashedWheelTimer(
```

Comment:
Pealse use 'private static final' since we do not need to expose it to other classes.

And please shutdown it in afterClass method.

Reply:
@Divneet18 please take care of this and you can also start preparing branch-2 PR after addressing this.